CODE_OF_CONDUCT.md

```diff
@@ -61,7 +61,8 @@ representative at an online or offline event.
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
 reported to the community leaders responsible for enforcement at
-[INSERT CONTACT METHOD].
+matplotlib@numfocus.org (monitored by Thomas Caswell and Hannah Aizenman), or
+senior NumFOCUS staff at conduct@numfocus.org.
```

Comment:
the NumFocus anonymous report form is  https://numfocus.typeform.com/to/ynjGdT

 conduct@numfocus.org is the moderator that issues the council cwe don't have can't resolve goes to & the important bit there is probably this link https://numfocus.org/code-of-conduct since it has the people on their CoC board. 

Also Thomas Caswell [@tacaswell] & Hannah Aizenman [@story645 ]

Reply:
I'm not sure I understood everything above, but is this now better/fixed?

Reply:
Reports get escalated to senior NumFOCUS staff at conduct@numfocus.org if they can't be resolved, but initial reporting should either be to matplotlib@numfocus.org or the typeform.

Reply:
I think it is OK if people want to email conduct@numfocus.org first in cases where I am the problem or they don't trust the matplotlib@numfocus.org address for what ever reason.